.github/workflows/pkgdown.yaml

```diff
@@ -39,8 +39,7 @@ jobs:
 
       - name: Deploy to GitHub pages 🚀
         if: github.event_name != 'pull_request'
-        uses: JamesIves/github-pages-deploy-action@4.1.4
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
Rather than doing this we should really be deploying as an environment.

Reply:
Something like this? https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#using-an-environment

Shall I merge it now or would you suggest we do that first?

Reply:
I don't mind. Maybe just make an issue to do it?